src/index.js

```diff
@@ -164,12 +164,22 @@ window.TurbolinksAnimate = window.TurbolinksAnimate || new function() {
         return;
       }
 
-      newElement.style[cssPropertyToCamelCase(property)] = getComputedStyle(element).getPropertyValue(property);
+      let properties = property.split(',');
```

Comment:
We need to document this in the readme.

Reply:
Good point, I can propose a change to the readme.

Reply:
https://github.com/jonhue/turbolinks-animate/pull/38/commits/350239020093b44fea937970f60158458b7e2a20